packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -132,7 +132,10 @@ export async function createSummarizerWithContainer(
     const loader = provider.makeTestLoader(testContainerConfig);
     return createSummarizerCore(absoluteUrl, loader, summaryVersion);
 }
-
+/**
+ * Summarizes on demand and returns the summary tree, the version number and the reference sequence number of the
+ * submitted summary.
```

Comment:
Maybe note that if a user wants a summary with all the ops summited to call `provider.ensureSynchronized()`